sympy/printing/pretty/pretty.py

```diff
@@ -1471,6 +1471,12 @@ def _print_Union(self, u):
         return self._print_seq(u.args, None, None, union_delimiter,
              parenthesize=lambda set: set.is_ProductSet or set.is_Intersection)
 
+    def _print_SymmetricDifference(self, u):
+
+        sym_delimeter = ' %s ' % pretty_atom('SymmetricDifference')
+
+        return self._print_seq(u.args, None, None, sym_delimeter)
```

Comment:
@hargup I didn't understand how `parenthesize` is used, so I haven't added anything there.